common-features/icons.md

```diff
@@ -33,8 +33,11 @@ This article contains the following sections:
 * [Complete list of built-in icons](#icons-list)
 * [How to use custom icons](#custom-icon-support)
 
+{% if site.has_cta_panels == true %}
+{% include cta-panel-introduction.html %}
+{% endif %}
 
-## How Icons Work
+## How Blazor Icons Work
```

Comment:
Also change the link in the bullet list above: `[How do icons work](#how-blazor-icons-work)`